.github/actions/setup/action.yml

```
@@ -5,16 +5,23 @@ inputs:
   node-version:
     description: 'Node.js version to install'
     default: '18'
+  pnpm-version:
+    description: 'pnpm version to install'
+    default: 8
 
 runs:
   using: 'composite'
   steps:
-    - name: 'Install Node.js'
+    - name: 'Install pnpm ${{ inputs.pnpm-version }}'
+      uses: 'pnpm/action-setup@v2'
```

Following GH Actions security best practices, we should pin this to a specific action commit

```suggestion
      uses: pnpm/action-setup@ c3b53f6a16e57305370b4ae5a540c2077a1d50dd # v2.2.4
```

---

Done. Did the same for the node setup action.

---

I tend to be a little more relaxed with the GitHub maintained actions, but certainly no harm in locking! I'd rather we lock everything than nothing